dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -22,15 +23,31 @@ def gallery
       to_json
   end
 
+  # Hard-coded list of channel IDs for the mini player if DCDO flag is set to `true`
+  CHANNELS = %w(
```

Comment:
Nit: move up with other constants, rename to something more specific (`FALLBACK_FEATURED_CHANNELS`)?